### PR TITLE
Includes are now packaged in conda builds. 

### PR DESCRIPTION
**Pull request by Kelvinrr**
## Description

allows for headers to be installed during conda build

## Related Issue

https://github.com/USGS-Astrogeology/ISIS3/issues/3892

## How Has This Been Tested?

I ran `conda build .` locally (with meta.yml to point to my changes), unpacking the tar had `includes/isis/*.h` files when they previously didn't exist. 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation change (update to the documentation; no code change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have read and agree to abide by the [Code of Conduct](https://usgs-astrogeology.github.io/code/)
- [ ] I have read the [**CONTRIBUTING**](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] I have added myself to the [AUTHORS.rst](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/AUTHORS.rst) document.

## Licensing
This project is mostly composed of free and unencumbered software released into the public domain, and we are unlikely to accept contributions that are not also released into the public domain. Somewhere near the top of each file should have these words:

> This work is free and unencumbered software released into the public domain. In jurisdictions that recognize copyright laws, the author or authors of this software dedicate any and all copyright interest in the software to the public domain.

- [ ] I dedicate any and all copyright interest in this software to the public domain. I make this dedication for the benefit of the public at large and to the detriment of my heirs and successors. I intend this dedication to be an overt act of relinquishment in perpetuity of all present and future rights to this software under copyright law.

### CODE REVIEW

File: isis/CMakeLists.txt [hunk 1]
```diff
@@ -428,9 +428,9 @@ EXECUTE_PROCESS(COMMAND cp -f ${CMAKE_SOURCE_DIR}/TestPreferences ${CMAKE_BINARY
 install(CODE "EXECUTE_PROCESS(COMMAND cp -f ${CMAKE_SOURCE_DIR}/IsisPreferences ${CMAKE_INSTALL_PREFIX}/)")
 
 # Install the header files
-install(CODE "EXECUTE_PROCESS(COMMAND mkdir -p ${CMAKE_INSTALL_PREFIX}/include/isis3)")
+install(CODE "EXECUTE_PROCESS(COMMAND mkdir -p ${CMAKE_INSTALL_PREFIX}/include/isis)")
 file(GLOB ISIS_INCLUDE_FILES ${CMAKE_BINARY_DIR}/inc/*)
-install(CODE "EXECUTE_PROCESS(COMMAND cp -fv ${ISIS_INCLUDE_FILES} ${CMAKE_INSTALL_PREFIX}/include/isis3)")
+install(DIRECTORY ${CMAKE_BINARY_DIR}/inc/ DESTINATION  ${CMAKE_INSTALL_PREFIX}/include/isis)
```
Comment (Kelvinrr) [inline isis/CMakeLists.txt:433]: For some reason this copy command wasn't being executed on conda build, replacing it with a proper install command seemed to fix it.

File: isis/CMakeLists.txt [hunk 2]
```diff
@@ -428,9 +428,9 @@ EXECUTE_PROCESS(COMMAND cp -f ${CMAKE_SOURCE_DIR}/TestPreferences ${CMAKE_BINARY
 install(CODE "EXECUTE_PROCESS(COMMAND cp -f ${CMAKE_SOURCE_DIR}/IsisPreferences ${CMAKE_INSTALL_PREFIX}/)")
 
 # Install the header files
-install(CODE "EXECUTE_PROCESS(COMMAND mkdir -p ${CMAKE_INSTALL_PREFIX}/include/isis3)")
+install(CODE "EXECUTE_PROCESS(COMMAND mkdir -p ${CMAKE_INSTALL_PREFIX}/include/isis)")
```
Comment (Kelvinrr) [inline isis/CMakeLists.txt:431]: I imagine since ISIS headers were not being shipped to begin with, renaming the header location to isis is okay.

**Review by jessemapel — APPROVED** [review]

File: isis/CMakeLists.txt (hunk 1 above)
Comment (jessemapel) [reply isis/CMakeLists.txt:433]: We have several places were we use shell cp to install things. Proper install calls are something that should eventually be used everywhere.

File: isis/CMakeLists.txt (hunk 2 above)
Comment (krlberry) [reply isis/CMakeLists.txt:431]: Agreed, and this seems like a great time to make the change from isis3 to isis!

**Review by krlberry — APPROVED** [review]
:+1: Looks good!

**Comment by jessemapel** [comment]
Fixes #3892

**Pull request merged**